woocommerce/woocommerce
# Mirror Core to production repo

### All Submissions:






### Changes proposed in this Pull Request:

Commits merged to `trunk` and release branches will be automatically copied to the [production read-only repo](https://github.com/woocommerce/woocommerce-production). 

Fixes https://github.com/woocommerce/woocommerce/issues/31062

Note: this depends on https://github.com/woocommerce/action-build/pull/6

### How to test the changes in this Pull Request:

The Github Action is temporarily set to run on `push` to `test/build-prod-repo-on-push` so that we can test using changes to this PR instead of `trunk`. After review and before final merge, the branches in question will be updated to `trunk` and `release/**`.

1. See the `Mirrors` Github Action run without error.
2. Look at the [production mirror](https://github.com/woocommerce/woocommerce-production/tree/test/build-prod-repo-on-push) for core and confirm WooCommerce is built and represented.
3. Check the commit history is accurate.

### Other information:




### Changelog entry

Copy WooCommerce Core to production read only repository on changes made to trunk or release branches.

### FOR PR REVIEWER ONLY:
